### DISCUSSION TITLE
Aqara Z1 (ZNQBKG38LM)

### DISCUSSION THREAD

**Discussion by geertschneider** [Koenkk/zigbee2mqtt#24146] [Devices]
The Aqara Z1 allows tap / double tap / hold functionality.

This is visible in the click_mode having 2 possible values: fast, multi.
On the [Action (enum)](https://www.zigbee2mqtt.io/devices/ZNQBKG38LM.html#action-enum) however there is only 1 value: **single**

How could we detect if there was a double tap ?
I see on other devices it exposes actions as : 'single', 'double', 'hold'

**Comment by guidobbb** [comment]
@Koenkk Koen, can this be added? Do you need more info?

**Comment by guidobbb** [comment]
@geertschneider Geert, how does the click 'feel'? is it a bit nice, or 'too plastic'?
dank je.

**Comment by geertschneider** [comment]
I like the device a lot.
What it would make it even better is a doubleclick / or long tap.
